ui/components/chat-area.jsx

```diff
@@ -41,7 +42,7 @@ module.exports = function(core, config, store) {
 						<div className="chat-area-actions">
 							<ScrollTo type="bottom" onClick={this.scrollToBottom} />
 
-							<Compose />
```

Comment:
Removed <GoToThread>

Satyajit Sahoo
UX Designer
Behance Profile <https://www.behance.net/satyajitha28c7>

We're all stories, in the end. Just make it a good one, eh? — The Doctor,
Season 5, Episode 13.

On 10 September 2015 at 14:57, Aravind <notifications@github.com> wrote:

> In ui/components/chat-area.jsx
> <https://github.com/scrollback/scrollback/pull/892#discussion_r39140495>:
>
> > @@ -41,7 +42,7 @@ module.exports = function(core, config, store) {
> >  						<div className="chat-area-actions">
> >  							<ScrollTo type="bottom" onClick={this.scrollToBottom} />
> >
> > -							<Compose />
>
> no, we cant change behavior for the irc guys.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scrollback/scrollback/pull/892/files#r39140495>.
>